prow/jobs/kyma-project/test-infra/images.yaml

```diff
@@ -1,5 +1,52 @@
 presubmits: # runs on PRs
   kyma-project/test-infra:
+    - name: pull-build-image-builder
```

Comment:
You can add the go application to .koapps.yaml and in .ko.yaml under baseImageOvverides: add line
```
-  github.com/kyma-project/test-infra/cmd/image-builder: gcr.io/kaniko-project/executor:v1.14.0

```
This will simplify the configuration and will not require from you to add yet another ProwJob we wanted to reduce number, and completely drop the need for Dockerfile

Reply:
For now I prefer to stay with Dockerfile. It works fine and will let me build in ADO in the future.

Reply:
Look, we still use Dockerfiles and I prefer to use it.
I don't remember that we agreed I can't use a dockerfile anymore.
image-builder is build using a dockerfile. I just want to make it automated and pushed to the production registry.
Having in mind upcoming changes in CI/CD I don't want to change a way how image-builder was build.

Reply:
@akiioto please move your comment and discussion to the dedicated issue.